atlasdb-dbkvs/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/TableValueStyleCacheTest.java

```diff
@@ -45,6 +47,9 @@
     private final ConnectionSupplier connectionSupplier = mock(ConnectionSupplier.class);
     private final TableValueStyleCache valueStyleCache = new TableValueStyleCache();
 
+    @Rule
+    public MetricsRule metricsRule = new MetricsRule();
```

Comment:
This one is not instrumented, right? We dont need the ``MetricsRule`` here or we can check that no metrics are registered in the tests.